release-notes/opensearch-release-notes-2.19.3.md

```diff
@@ -0,0 +1,219 @@
+# OpenSearch and OpenSearch Dashboards 2.19.3 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 2.19.3](https://opensearch.org/versions/opensearch-2-19-3.html) includes the following bug fixes, infrastructure, documentation and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/2.19/release-notes/opensearch.release-notes-2.19.3.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.19/release-notes/opensearch-dashboards.release-notes-2.19.3.md).
+
+
+## ENHANCEMENTS
+
+
+### Opensearch Flow Framework
+
+
+* Fix ApiSpecFetcher Memory Issues and Exception Handling ([#1192](https://github.com/opensearch-project/flow-framework/pull/1192))
+
+
+* Better handling of Workflow Steps with Bad Request status ([#1191](https://github.com/opensearch-project/flow-framework/pull/1191))
```

Comment:
Patch releases should have an enhancements. Check with the owners for right category please.

Reply:
Which category do you recommend, will change it for the consolidated ones, component can be tracked separately.

Reply:
@ohltyler @owaiskazi @dbwiddis Can you guys recommend? I believe `bug fix` and `refactoring`  should be the okay. Please correct if I am wrong. 
Thanks!

Reply:
Bug fix